tailfs/tailfsimpl/dirfs/removeall.go

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package dirfs
+
+import (
+	"context"
+	"os"
+)
+
+// RemoveAll implements webdav.File. No removal is supported and this always
+// returns os.ErrPermission.
```

Comment:
Do we care to distinguish "not found" from "computer says no"? (Some tools care, but I'm not sure it matters that much)

Reply:
I don't think we care enough to make it worth adding that code here.